src/js/db/story-dbConfiguration.js

```diff
@@ -33,37 +33,94 @@ fluid.defaults("sjrk.storyTelling.server.storiesDb", {
         "storyExample": {
             "type": "story",
             "value": {
-                "title": "Using This Site",
+                "title": "The Story Builder how-to",
                 "content": [{
                     "id": null,
                     "language": null,
-                    "heading": "Using This Site - Part One",
+                    "heading": "Add Content Blocks",
                     "blockType": "text",
-                    "text": "Click the icons above to add blocks of various types, such as text and images, to the story.",
+                    "text": "The Story Builder is designed based on building blocks. There are four types of blocks you can use to build your story:\n\n1. Written content—type directly in the tool\n2. images (photos of items, drawings, diagrams, etc.)—capture photos or upload files\n3. audio (sound)—record audio or upload files\n4. video (audio and visual)—record video or upload files\n\nTo add content blocks and start creating your story, click on an icon at the top of the Story Builder to add a block. You can add as many blocks as you like.\n\nYou won’t be able to reorder the blocks once you have added them, thus, it may be helpful to outline your story blocks before you add them in the Story Builder Tool.",
```

Comment:
Shall first letters after list numbers be capitalized? images -> Images, audio -> Audio, video -> Video

Same for these in "contentString" value.